tests/ignite/metrics/test_rouge.py

```diff
@@ -0,0 +1,195 @@
+import os
+
+import pytest
+import torch
+
+import ignite.distributed as idist
+from ignite.exceptions import NotComputableError
+from ignite.metrics import Rouge
+
+
+def test_zero_div():
+    rouge = Rouge()
+    with pytest.raises(NotComputableError):
+        rouge.compute()
+
+
+def test_input():
+    with pytest.raises(TypeError):
+        rouge = Rouge(beta="l", n=1)
+
+    with pytest.raises(ValueError):
+        rouge = Rouge(n="m")
+
+    with pytest.raises(ValueError):
+        rouge = Rouge(n=-1)
+
+
+def test_compute():
+    rouge = Rouge()
+
+    y_pred = "the cat was found under the bed"
+    y = "the cat was under the bed"
+    rouge.update([y_pred.split(), y.split()])
+    assert isinstance(rouge.compute(), torch.Tensor)
+    assert rouge.compute() == 0.8571428571428571
+
+    y_pred = "the tiny little cat was found under the big funny bed"
+    y = "the cat was under the bed"
+    rouge.update([y_pred.split(), y.split()])
+    assert isinstance(rouge.compute(), torch.Tensor)
+    assert rouge.compute() == 0.7012987012987013
+
+    rouge = Rouge(n="l")
+
+    y_pred = "the cat was found under the bed"
+    y = "the cat was not under the bed"
+    rouge.update([y_pred.split(), y.split()])
+
+    assert isinstance(rouge.compute(), torch.Tensor)
+    assert rouge.compute() == 0.8571428571428571
+
+    y_pred = "the cat was found under the big funny bed"
+    y = "the tiny little cat was under the bed"
+    rouge.update([y_pred.split(), y.split()])
+
+    assert isinstance(rouge.compute(), torch.Tensor)
+    assert rouge.compute() == 0.7619047619047619
+
+
+def _test_distrib_integration(device):
```

Comment:
I realized that this test does not check any numeric values.
This test should be refactor to be compatible with distributed validation.

The idea is to have a corpus replicated on each process. Each process work on a part of the corpus and compare the result of the metric with the local computation done.

See for instance https://github.com/pytorch/ignite/blob/455a3b3282eca7e15d45c22f7e4e384faef465fa/tests/ignite/metrics/test_precision.py#L784